### ISSUE TITLE
Redirect uri issue on SPAs

### ISSUE THREAD

**Issue by anon314271** [MrSwitch/hello.js#596]
Hello! 
**Edit**
After some digging I don't think it's possible and most suggestions let are to load a static .html file 

**TL;DR**
 How can I implement a redirect_uri like '#/callback' so hellojs doesnt hang 

I'm working on an SPA (using Vue) and have stumbled into a small issue.
Whenever I set the redirect uri to:

     redirect_uri: "#/callback"

Hellojs hangs and I'm unable to log in.

However setting it to 

     redirect_uri: "callback"

is fine and I log in, the popup to login leads to the redirect_uri page.

The issue is routes in an SPA begin with "#/...." and I'm wondering how I can implement a redirect_uri like this (i.e. "#/.....")

I've thought of two possible solutions:

1.  serving a static .html file when redirecting to /callback
2.  using history mode (a solution in Vue that removes the leading #)

Solution 1 ignores the issue altogether
Solution 2 works but changes how the app is served

I'm wondering if anyone has had any experience resolving this issue or if I've missed something really obvious?

Ps. please let me know if I can provide more info / how I can debug this further

Thanks :)

-- edit: spelling and clarity
